acra-censor/acra-censor_implementation.go

```diff
@@ -110,3 +120,19 @@ func (acraCensor *AcraCensor) HandleQuery(rawQuery string) error {
 	acraCensor.logger.Infof("Allowed query: '%s'", queryWithHiddenValues)
 	return nil
 }
+
+func (acraCensor *AcraCensor) saveQuery(rawQuery string) error {
+	openedFile, err := os.OpenFile(acraCensor.parseErrorsLogPath, os.O_APPEND|os.O_RDWR|os.O_CREATE, 0600)
+	if err != nil {
+		return err
+	}
+
+	defer openedFile.Close()
+
+	_, err = openedFile.WriteString(rawQuery + "\n")
```

Comment:
here we will have same problem which we had with querycapture handler where we try to buffer data before dumping to file to avoid slow write on each query. here we will have same problem with situation when client will use not supported queries. plus we don't need repeated queries, only unique. maybe would be good to write some `QueryWriter` (and use in querycapture if we can to avoid duplicated code) which will do this logic with buffered write and timeouts and store unique queries in memory to avoid repeated queries

Reply:
Ok, I will write additional handler for this (with functionality like QueryCapture)

Reply:
agree, it makes sense to allocate same functionality into some object lke `QueryWriter` and use it for both `QueryCapture` and here